Assets/Scripts/PlayerControlls.cs

```diff
@@ -85,4 +86,14 @@ void Update()
             AxisTurn();
         }
     }
+
+    public Vector3 getLastTurn()
```

Comment:
Can't we just get a transform.rotation instead? Or will it be used explicitly when interfacing externally with `PlayerControlls`?